### DISCUSSION TITLE
How to add provider :developer

### DISCUSSION THREAD

**Discussion by francescor** [janko/rodauth-omniauth#26] [General]
It will help beginners (It would have helped me last days ;) to add a note in README on how to enable the `:developer` provider
as in https://github.com/omniauth/omniauth but when using this gem `rodauth-omniauth`.

I've just realized now it's just one line in `app/misc/rodauth_main.rb`

```
omniauth_provider :developer if Rails.env.development?
```

with no need to touch `config/routes.rb` and create `app/controllers/sessions_controller.rb` since it is all handled automatically by your gem (wow!)

Then, of course, the login link

```
  <li><%= button_to "Login with Developer", rodauth.omniauth_request_path(:developer), method: :post, data: { turbo: false }, class: "btn btn-link p-0" %></li>
```

**Comment by francescor** [comment]
uh, yes, so just adding those two `developer` lines in README, as you did for `facebook, twitter, google_oauth2...`  

yes, I realize now it was supposed to be obvious, but believe me I did not get it :(

**Reply by janko** [reply]
I don't see a convenient place to add it, and I'm not really onboard that it's necessary, given that the `developer` strategy is setup exactly the same as all other strategies.
